index.html

```diff
@@ -85,88 +90,203 @@
-          series or that has expired (see <a href="#expiration">below</a>) with a 409 response.
-          That response MUST include an <code>Atomic-Invalid</code> header or headers with the
-          invalid identifier or identifiers.
+          A client adds to an atomic batch operation by including an
+          <code>Atomic-ID</code> request header. The value of this header is a
+          transaction identifier.
+        </p>
+        <p>
+          The effect of the request, if it has one, and if the request is
+          successful, MUST be visible to clients using the same transaction
+          identifier as the value of the <code>Atomic-ID</code> header, as
+          described here, until the series expires or is finished. The effect of
+          the request, if it has one, and if the request is successful, SHOULD
+          NOT be visible to clients not using that transaction identifier. The
+          effect of the request MUST NOT be durable unless and until the series
+          is successfully <a href="#commit">finished</a>.
+        </p>
+        <p>
+          If the request is successful, the server MUST include a
+          <code>Atomic-ID</code> response header whose value MUST be the current
+          transaction identifier.
+        </p>
+        <p>
+          If the transaction identified by the <code>Atomic-ID</code> header is
+          invalid for any reason, the server MUST NOT update any resources, and
+          it MUST respond with a <code>409 Conflict</code> HTTP status. The
+          server MAY also include more information about the reason for
+          rejecting the request in the response body.
         </p>
       </section>
 
       <section id="expiration">
         <h2>Expiration</h2>
         <p>
-          An implementation MAY enforce automatic expiration for an atomic series of requests.
-          If so, the response to any request made as part of the series MUST have a
-          <code>Atomic-Expires</code> header with the HTTP datetime at which the series will
-          expire. An implementation MAY extend the lifetime of a series upon the receipt of any
-          request specifying the unique identifier of that series in the <code>Atomic-ID</code>
-          header.
+          An implementation MAY enforce automatic expiration for an atomic
+          series of requests.  If so, the response to any request made as part
+          of the series MUST have a <code>Atomic-Expires</code> header with the
+          HTTP datetime at which the series will expire.
         </p>
+        <p>
+          An implementation SHOULD extend the lifetime of a series upon the
+          receipt of any request to resources within the scope of the
+          transaction.
+        </p>
+        <p>
+          An implementation SHOULD permit extending the lifetime of a series by
+          a <code>POST</code> request to the transaction identifier URI.
```

Comment:
If we are considering a more deliberate message to cause a `commit` to occur, then I would suggest that pattern also be used here.

Reply:
I am open to suggestions on what that message body would look like...

Reply:
I guess I don't imagine my current application passing along any extra info during a commit, so I'd be okay with the body for POST and PUT being a MAY rather than requiring it for both